views/github-error.hbs

```diff
@@ -1,21 +1,37 @@
-<!doctype html>
 <html>
   <head>
-    <meta charset="utf-8">
-    <meta http-equiv="X-UA-Compatible" content="IE=edge">
-    <meta name="ap-local-base-url" content="{{localBaseUrl}}">
+    <meta charset="utf-8" />
+    <meta http-equiv="X-UA-Compatible" content="IE=edge" />
+    <meta name="ap-local-base-url" content="{{localBaseUrl}}" />
     <title>{{title}}</title>
+    <link rel="stylesheet" href="/public/css/github-error.css" media="all" />
     <link rel="stylesheet" href="/public/primer/build.css" media="all" />
     <script src="/public/js/jquery.min.js" nonce="{{nonce}}"></script>
   </head>
-  <body class="bg-gray px-3 py-6">
-    <img class="centered small mb-6" src="https://octodex.github.com/images/supportcat.png" height="300"/>
-    <section class="mx-auto" style="max-width: 510px">
-      <h2 class="f2 text-center text-normal mb-2">Looks like something went wrong!</h2>
+  <body class="githubError">
+    <section class="githubError__container">
+      <img
+        class="githubError__errorImg"
+        src="/public/assets/error.png"
+        alt="Error"
+      />
 
-      <hr />
+      <h3 class="githubError__header">Looks like something went wrong!</h3>
 
-      <p class="text-gray text-center mb-4">We track these errors automatically, but if the problem persists feel free to <a href="https://support.atlassian.com/contact/#/?inquiry_category=technical_issues&is_cloud=true&product_key=third-party-product">contact us</a>. In the meantime, try adding an organization again in Jira.</p>
+      <p class="githubError__message">
+        We track these errors automatically, but if the problem persists feel
+        free to
+        <a
+          class="githubError__link"
+          href="https://support.atlassian.com/contact/#/?inquiry_category=technical_issues&is_cloud=true&product_key=third-party-product"
```

Comment:
(This is not the part of the PR, but I'll ask anyway): shouldn't it lead them to GitHub issues instead?